tests/integration/test_arrowflight_interface/test.py

```diff
@@ -0,0 +1,93 @@
+# coding: utf-8
+
+import os
+import pytest
+import pyarrow as pa
+import pyarrow.flight as flight
+
+from helpers.cluster import ClickHouseCluster, get_docker_compose_path
+from helpers.test_tools import TSV
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+DOCKER_COMPOSE_PATH = get_docker_compose_path()
+
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance(
+    "node",
+    main_configs=["configs/config.xml"],
+    user_configs=["configs/users.xml"],
+)
+
+@pytest.fixture(scope="module", autouse=True)
+def start_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_doput():
+    node.query(
+        """
+        CREATE TABLE doput_test (
+            id Int64,
+            name String
+        )
+        ORDER BY id
+        """
+    )
+    client = flight.FlightClient(f"grpc://{node.ip_address}:8888")
```

Comment:
Is it possible to implement some way to pass a ClickHouse username and a password when we connect to ClickHouse using Arrow Flight interface?

Reply:
it's possible. I will do this soon

Reply:
added auth and tls support.

Authorization may be not working without tls for secure reasons in some clients

Updated tests